bin/buildBlueprints.js

```diff
@@ -130,40 +131,52 @@ console.log('...Reading Blueprints', argv.blueprintsPath);
 console.log('...cwd', process.cwd());
 
 var config = makeConfig(argv);
+var isTest = argv.target === TARGETS.TEST;
 
 build(config, function(stats) {
   if (stats.errors && stats.errors.length > 0 && !argv.watch) {
     console.log(colors.red('ERROR IN BUILD. Aborting.'));
     process.exit(1);
   }
 
-  if (argv.runTest) {
+  if (isTest) {
     console.log(colors.magenta(
       '\n   ******************************' +
       '\n   *       RUNNING TESTS        *' +
       '\n   ******************************'
     ));
 
     m = new Mocha({ reporter: mochaNotifier.decorate('spec') });
```

Comment:
I'd also really like that and I believe I explored it to some degree and found it difficult or impossible (forget exactly which) to do this programmatically. Everything had to be done via CLI.